### PR TITLE
PSR-767 bonds list page

### PR DESCRIPTION
**Pull request by narendrapaduchuri**

### CODE REVIEW

**Review by kberec — DISMISSED** [review]

File: app/controllers/testonly/BondsMongoController.scala [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+import pages.nonsipp.unregulatedorconnectedbonds._
+import play.api.mvc.MessagesControllerComponents
+import services.SaveService
+import shapeless._
+
+import java.time.LocalDate
+import javax.inject.Inject
+import scala.concurrent.ExecutionContext
+
+class BondsMongoController @Inject()(
```
Comment (kberec) [inline app/controllers/testonly/BondsMongoController.scala:33]: I thought this is in @westwater 's  branch @narendrapaduchuri ?

File: .g8/textInputPage/app/pages/$className$Page.scala [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+import models.SchemeId.Srn
+import models.Money
+$if(!index.empty)$
+import config.Refined.$index$
+import utils.RefinedUtils._
+import eu.timepit.refined.refineMV
+$endif$
+
+$if(index.empty)$
+case class $className;format="cap"$Page(srn: Srn) extends QuestionPage[String] {
```
Comment (kberec) [inline .g8/textInputPage/app/pages/$className$Page.scala:13]: Are these changes intentional @narendrapaduchuri ?

File: app/controllers/testonly/BondsMongoController.scala (hunk 1 above)
Comment (narendrapaduchuri) [reply app/controllers/testonly/BondsMongoController.scala:33]: Yes @kberec , George also added this end point.. I will wait until George ticket merge to main

File: .g8/textInputPage/app/pages/$className$Page.scala (hunk 2 above)
Comment (narendrapaduchuri) [reply .g8/textInputPage/app/pages/$className$Page.scala:13]: I think this is system generated ones, I haven't added that

**Review by kberec — APPROVED** [review]

**Review by kberec — APPROVED** [review]

**Pull request merged**